README.md

```diff
@@ -25,3 +25,13 @@ More details about the icons, their installation, usage etc. could be found in t
 
 * SVG Icons - https://www.telerik.com/design-system/docs/foundation/iconography/svg-icons/
 
+## SVG creation - the design flow:
+- Create your icon in Adobe Illustrator and follow the guidelines described in https://www.telerik.com/design-system/docs/foundation/iconography/svg-icons/
```

Comment:
The url is incorrect - it should point to iconography -> Style and Guidelines. I.e.:
https://www.telerik.com/design-system/docs/foundation/iconography/styles-and-guidelines/